src/patterns/OcFileActions.vue

```diff
@@ -53,6 +55,15 @@ export default {
       required: false,
       default: "",
     },
+    /**
+     * The actions from which the user can select
+     */
+    items: {
+      type: Array,
```

Comment:
required: true maybe?

Reply:
more interesting: how do we pass this in when clicking on a file in the file template?

Reply:
vue glue code in the files app

Reply:
props are bound from the parent component